### PR TITLE
[WIP] Show the component status within a label

### PR DESCRIPTION
**Pull request by jbrooksuk**
Looks like this:

image

A custom stylesheet can be applied to remove the colours though.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Moved the milestone. I assume you clicked the wrong one by mistake?

**Comment by jbrooksuk** [comment]
I did.

**Comment by GrahamCampbell** [comment]
Let me know when you're ready to start merging "features", and I'll spin off a 2.1 branch first.

**Comment by GrahamCampbell** [comment]
Would be useful to make this configurable?

**Comment by TheoBearman** [comment]
I think so.

**Comment by jbrooksuk** [comment]
Would it? You can just re-style them.

File: app/Presenters/ComponentPresenter.php [hunk 1]
```diff
@@ -35,6 +35,21 @@ public function status_color()
...
 
     /**
+     * Return the color for the status label.
+     *
+     * @return string
+     */
+    public function label_color()
+    {
+        switch ($this->wrappedObject->status) {
+            case 1: return 'success';
```
Comment (GrahamCampbell) [inline app/Presenters/ComponentPresenter.php]: cs

Comment (GrahamCampbell) [inline app/Presenters/ComponentPresenter.php]: we don't yet have a fixer for this on styleci, but currently, this isn't PSR-2

**Pull request closed without merging**